worker/pyproject.toml

```diff
@@ -29,6 +29,13 @@ deploy = [
     "gunicorn>=21.2.0",
 ]
 
+dev = [
```

Comment:
```suggestion
test = [
```

Reply:
Added it as dev since test related items are usually added as dev dependencies. Will keep it as the same for now.